mobile/src/root.tsx

```diff
@@ -21,11 +20,6 @@ export class Root extends Component<{}> {
         config.oauth.redirectUri);
 
       this.navigationService = new NavigationService();
-
-      const isIOS = Platform.OS === 'ios';
```

Comment:
does it still work on Android? I don't remember the code but from the looks of it, something else caused Android app to login, now you call it for both?

Reply:
Yes, now the login procedure is started in App component for both platforms.